### DISCUSSION TITLE
How to generate flame graph

### DISCUSSION THREAD

**Discussion by J1a-wei** [Kong/kong#10216] [Help] [answered]
We have developed the Kong plugin ourselves. 
How to see whether there is a performance bottleneck and whether there is any way to help us judge the performance of this plugin. 
For example, the flame graph should preferably show the function frame of the plugin.

**Comment by ADD-SP** [comment, marked as answer]
Hello @0xRook1e, usually, the flamegraph is the best way to analyze the performance issue, you can read this [docs](https://github.com/Kong/stapxx#lj-lua-stacks), but we do not guarantee that these tools always work.
